Fields.sc

```diff
@@ -1,157 +1,157 @@
-// �2009 Miguel Negr�o
+// 2009 Miguel Negro
```

Comment:
Please change to "Copyright 2009 Miguel Negrão"

Reply:
See above